### ISSUE TITLE
`_refocusOnMap` forces keyboard/swipe gesture users to navigate all the way back to continue interacting with a control + causes verbose speech

### ISSUE THREAD

**Issue by danallsop** [ScottLogic/jumpstart-Leaflet#334]
> **Migrated from:** https://github.com/Leaflet/Leaflet/issues/7947
> **Original author:** danallsop
> **Original created:** 2022-01-26T10:57:33Z
---

- [x] I've looked at the [documentation](http://leafletjs.com/reference.html) to make sure the behavior is documented and expected
- [x] I'm sure this is a Leaflet code issue, not an issue with my own code nor with the framework I'm using (Cordova, Ionic, Angular, React…)
- [x] I've searched through the issues to make sure it's not yet reported

**Steps to reproduce**

1. Go to https://leafletjs.com/examples/quick-start/example.html
2. Turn on the [screen reader](https://leafletjs.com/examples/accessibility/#test-your-maps) of your choice
3. <kbd>Tab</kbd> (or swipe) to the zoom in control
4. Press <kbd>Enter</kbd> (or double tap)

**Expected behavior**

- The user should be able to continuously zoom in/out
- Screen readers should not announce everything in the map repeatedly on zoom

**Current behavior**

- The user must tab/swipe through all markers again to continue zooming
- Screen reader announcement is too verbose (announces everything a second time - including attribution and controls - after zoom)

(You may have to unmute the video:)

https://user-images.githubusercontent.com/26493779/151146426-4fe45c18-97b6-4745-8f70-abc749b30601.mp4

**Environment**
- Leaflet version: All versions since https://github.com/Leaflet/Leaflet/pull/2150.
- Browser (with version): Chrome v97.0.4692.87
- OS/Platform (with version): Android v12
- Assistive technology: TalkBack v13.0.02.0

**Additional context**

- The current behavior fails WCAG 2.1 [Success Criterion 3.2.2 On Input](https://www.w3.org/TR/WCAG21/#on-input).
- The problem for end-users (who use screen readers) is pretty much the same as https://github.com/Leaflet/Leaflet/issues/8114
- https://github.com/Leaflet/Leaflet/issues/3159 was resolved by https://github.com/Leaflet/Leaflet/pull/3275, but that solution is seemingly not sufficient.
- https://github.com/Leaflet/Leaflet/issues/4125#issuecomment-428800176:

  > Another possible solution would be to make it possible to turn off Leaflet refocusing on map, maybe through a control or map option, or a global flag.

I believe https://github.com/Leaflet/Leaflet/pull/2150 would have to be reverted, or the default behavior changed at least.

/cc danallsop, danallsop, danallsop

**Comment by danallsop** [comment]
## Comments (1 total)

---
> **Comment by:** danallsop **on** 2022-03-22T16:50:01Z

Changes to the default focus behavior generally causes issues such as described in this issue.

Is it acceptable to simply remove `_refocusOnMap`?
